src/ensembl/src/shared/components/external-link/ExternalLink.scss

```diff
@@ -12,11 +12,20 @@
   top: 1px;
   position: relative;
   height: 12px;
+  width: 12px;
 }
 
 .link {
   display: inline-block;
   grid-column: link;
   position: relative;
   top: -2px;
+  margin-left: 5px;
+}
+
+.noLink {
+  display: inline-block;
+  grid-column: nolink;
+  position: relative;
```

Comment:
These three CSS properties seem to make no difference to the styling when I comment them out. Do we need these?

Reply:
I've removed the unused styles